exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -373,6 +384,9 @@ func addSingleHistogramDataPoint(pt pdata.HistogramDataPoint, resource pdata.Res
 			Value:     float64(cumulativeCount),
 			Timestamp: time,
 		}
+		if pt.Flags().HasFlag(pdata.MetricDataPointFlagNoRecordedValue) {
+			bucket.Value = math.Float64frombits(value.StaleNaN)
+		}
```

Comment:
Is this the protocol in Prometheus that you do this for all the buckets?

Reply:
I'm not sure whether it is documented, but the observed behavior is that the stale marker is produced for individual buckets that disappear.  Thus I would expect that when the entire histogram disappears each bucket should be marked stale.

Reply:
@Aneurysm9 FYI you need a followup on our proto/specification to ensure that we send the buckets definition when we set the flag as well, because this was not clear to me at least.